internal/metrics/workload.go

```diff
@@ -68,6 +68,11 @@ func (wrkM *WorkloadMetrics) WorkloadStarted(workloadName string, report []*podm
 			continue
 		}
 
+		var filter SampleFilter = &PermissiveAllowList{}
+		if cfg.Metrics.AllowList != nil && len(cfg.Metrics.AllowList.Names) > 0 {
```

Comment:
@jakub-dzon  not sure here, if there are no Names defined should we do Restrictive or Permissive.

Reply:
IMO when it's nil, it should be Permissive, if empty - Restrictive.

Reply:
:+1:  I was not sure about.